src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/Common/DbConnectionStringCommon.cs

```diff
@@ -524,6 +601,7 @@ internal static partial class DbConnectionStringDefaults
         internal static readonly SqlAuthenticationMethod Authentication = SqlAuthenticationMethod.NotSpecified;
         internal const SqlConnectionColumnEncryptionSetting ColumnEncryptionSetting = SqlConnectionColumnEncryptionSetting.Disabled;
         internal const string EnclaveAttestationUrl = "";
+        internal static readonly SqlConnectionAttestationProtocol AttestationProtocol = SqlConnectionAttestationProtocol.NotSpecified;
```

Comment:
Can this be const? as static readonly it will be initialized when the class is loaded and can never be changed but because it isn't const the value must always be fetched on use. Making it const would allow the jit to inline the value instead.

Reply:
Good point. I will change it to `const`.